core/src/main/java/org/apache/iceberg/BaseTableScan.java

```diff
@@ -176,11 +188,26 @@ public Expression filter() {
 
   @Override
   public CloseableIterable<CombinedScanTask> planTasks() {
-    long splitSize = targetSplitSize(ops);
-    int lookback = ops.current().propertyAsInt(
-        TableProperties.SPLIT_LOOKBACK, TableProperties.SPLIT_LOOKBACK_DEFAULT);
-    long openFileCost = ops.current().propertyAsLong(
-        TableProperties.SPLIT_OPEN_FILE_COST, TableProperties.SPLIT_OPEN_FILE_COST_DEFAULT);
+    long splitSize;
+    if (splitOptions != null && splitOptions.getSplitSize() != null) {
+      splitSize = splitOptions.getSplitSize();
+    } else {
+      splitSize = targetSplitSize(ops);
+    }
+    int lookback;
+    if (splitOptions != null && splitOptions.getSplitLookback() != null) {
+      lookback = splitOptions.getSplitLookback();
+    } else {
+      lookback = ops.current().propertyAsInt(
+          TableProperties.SPLIT_LOOKBACK, TableProperties.SPLIT_LOOKBACK_DEFAULT);
```

Comment:
On the write path, we decided to go with easier to remember option names, like "snapshot-id" and "write-format", to be less verbose. What do you think about doing that here? This could be "lookback", "split-size", and "file-open-cost".

Reply:
Actually, "snapshot-id" is a read option, not a write option: http://iceberg.apache.org/configuration/#read-options

Not sure how I missed that. Anyway, I think we've set a precedent that we should use shorter option names for interactive cases like this, instead of full config names.

Could you also update the read options table?

Reply:
Do you mean to not accept, say, `read.split.target-size` in favor of `split-size`, or that  we should accept both?

Re updating the docs, will do.

Reply:
Went with short option names for the Spark `Reader`, but full property name when passed to the `TableScan` object. Let me know if this matches the precedent?

Reply:
I think accepting both is a good idea